spec/windows/block_device.go

```diff
@@ -7,26 +7,31 @@ import (
 	"unsafe"
 
 	"github.com/mackerelio/mackerel-agent/logging"
-	. "github.com/mackerelio/mackerel-agent/util/windows"
+	"github.com/mackerelio/mackerel-agent/util/windows"
 )
 
+// BlockDeviceGenerator XXX
 type BlockDeviceGenerator struct {
 }
 
+// Key XXX
 func (g *BlockDeviceGenerator) Key() string {
 	return "block_device"
 }
 
 var blockDeviceLogger = logging.GetLogger("spec.block_device")
 
+// Generate XXX
 func (g *BlockDeviceGenerator) Generate() (interface{}, error) {
 	results := make(map[string]map[string]interface{})
 
 	drivebuf := make([]byte, 256)
-	_, r, err := GetLogicalDriveStrings.Call(
+	r, _, err := windows.GetLogicalDriveStrings.Call(
 		uintptr(len(drivebuf)),
 		uintptr(unsafe.Pointer(&drivebuf[0])))
-	if r != 0 {
+	if r == 0 {
```

Comment:
[IMO] I'd like to rename this variable to other than `r` because this return value is set to 0 on errors, unlike others.